### ISSUE TITLE
'Fingerprinting Protection' label occludes toggle for Italian users

### ISSUE THREAD

**Issue by jonathansampson** [brave/browser-ios#1443]
### Description:
When using the Italian language, users may not be able to toggle _Fingerprinting Protection_ on and off.

**Reproduces how often:** [Easily reproduced/Intermittent issue]
Always

### Additional Information

![dwapwxyxkautwzr](https://user-images.githubusercontent.com/815158/36216794-652ad0e8-1175-11e8-9f87-aa2596c92709.jpg)

**Comment by srirambv** [comment]
I think the screenshot is for new tab. It will show enabled if the global setting is enabled. Its the same for all languages.

**Comment by iccub** [comment]
I think we could do a better job at pointing users to a place where they could adjust global settings for blockers.

**Comment by jonathansampson** [comment]
@iccub This isn't about global settings; this is about the label overlapping the button.

**Comment by iccub** [comment]
@jonathansampson as Sriram said, on this screen, all these switches are disabled and can't be interacted with. They are just to show what global settings were enabled, and user needs to go to app settings in order to change them. For me this is just bad ux

**Comment by jonathansampson** [comment]
@iccub That is correct; I've asked the user to visit a website to see if the interaction is possible. At the very least, I think you're right about this being a UX/UI issue.
